jte-spring-boot-starter-3/src/main/java/gg/jte/springframework/boot/autoconfigure/JteAutoConfiguration.java

```diff
@@ -34,17 +35,15 @@ public JteViewResolver jteViewResolver(TemplateEngine templateEngine) {
     @Bean
     @ConditionalOnMissingBean(TemplateEngine.class)
     public TemplateEngine jteTemplateEngine() {
-        if(jteProperties.usePreCompiledTemplates()){
+        if (Boolean.TRUE.equals(jteProperties.usePreCompiledTemplates())) {
```

Comment:
Why are you boxing `boolean` here?

This should just be `if(jteProperties.usePreCompiledTemplates())`

Reply:
SonarLint screamed at me to that, and as I'm normally using Kotlin I thought there must be a good reason to do so